tests/manage/z_cluster/test_add_mds_to_cluster.py

```diff
@@ -72,6 +72,7 @@ def get_mds_active_count():
 
 
 # @tier1
+# @skipif_external_mode
```

Comment:
Do we need this here if the test case is already disabled?

Reply:
Not necessarily, but since the test itself was not deleted and we are hoping this will be supported in future for internal mode, I added the skip marker :)